### ISSUE TITLE
windows vcpkg build fail

### ISSUE THREAD

**Issue by hz-bin** [etcd-cpp-apiv3/etcd-cpp-apiv3#149]
CONTROL file:
```
Source: etcd-cpp-apiv3
Version: 0.2.6
Homepage: https://github.com/etcd-cpp-apiv3/etcd-cpp-apiv3
Description: The etcd-cpp-apiv3 is a C++ API for etcd's v3 client API, i.e., ETCDCTL_API=3.
Build-Depends: boost-asio, boost-system, boost-thread, boost-random,
               cpprestsdk, grpc, openssl, protobuf
```

portfile.cmake:
```
vcpkg_from_github(
    OUT_SOURCE_PATH SOURCE_PATH
    REPO etcd-cpp-apiv3/etcd-cpp-apiv3
    REF f21c45b362297131b191ea8fc5033f8ffca4cc04
    SHA512 26ae637950b0d8d1527cce4f7a94aa0dce69598185c4e2040990e11ffdbe699b691be648b87537084c3503575baa65a52521659e160b27cce014c60406242d46
    HEAD_REF master
)

vcpkg_configure_cmake(
    SOURCE_PATH ${SOURCE_PATH}
    PREFER_NINJA
    OPTIONS
        -DBUILD_ETCD_TESTS=OFF
)
set(VCPKG_POLICY_DLLS_WITHOUT_LIBS enabled)
set(VCPKG_POLICY_DLLS_WITHOUT_EXPORTS enabled)

vcpkg_install_cmake()
vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/etcd-cpp-api)

file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)

vcpkg_copy_pdbs()

# Handle copyright
file(INSTALL ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/etcd-cpp-apiv3 RENAME copyright)
```

build error log:
[install-x64-windows-dbg-out.log](https://github.com/etcd-cpp-apiv3/etcd-cpp-apiv3/files/9299682/install-x64-windows-dbg-out.log)

**Comment by hz-bin** [comment]
$ vcpkg --version
vcpkg package management program version 2022-07-21-a0e87e227afb536c62188c11ad029954f28fdb22

**Comment by sighingnow** [comment]
It was a compatibility issue on Windows, see also: https://stackoverflow.com/questions/2561368/illegal-token-on-right-side-of

**Comment by sighingnow** [comment]
The patch has been included in the latest release v0.2.7 and the issue should be resolved in that release.

**Comment by hz-bin** [comment]
大佬666！！！
